<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QDockWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>240</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(181, 181, 181);</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Terrain Controls/Render</string>
    </property>
    <widget class="QLabel" name="terrainPaint_sizeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size: 5%</string>
     </property>
    </widget>
    <widget class="QLabel" name="terrainPaint_changeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QSlider" name="terrainPaint_sizeSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>81</width>
       <height>15</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>25</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="terrainPaint_changeSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="render_2DroadsStrokeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stroke: 3.5</string>
     </property>
    </widget>
    <widget class="QSlider" name="render_2DroadsStrokeSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>35</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="render_2DroadsExtraWidthSlider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="render_2DroadsExtraWidthLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R Width: 0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="render_2DparksLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Park 10 %</string>
     </property>
    </widget>
    <widget class="QSlider" name="render_2DparksSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="render_3DtreesCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3D Trees</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="terrain_smooth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Smooth Terrain</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>terrainPaint_sizeSlider</tabstop>
  <tabstop>terrainPaint_changeSlider</tabstop>
  <tabstop>render_3DtreesCheckBox</tabstop>
  <tabstop>render_2DroadsExtraWidthSlider</tabstop>
  <tabstop>render_2DroadsStrokeSlider</tabstop>
  <tabstop>render_2DparksSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
